.mergify.yml

```diff
@@ -37,3 +37,12 @@ pull_request_rules:
         branches:
           - "7.12"
         title: "[7.12] {{ title }} (backport #{{ number }})"
+  - name: Automatic squash and merge with success checks and the file docker-compose.yml is modified.
+    conditions:
+      - check-success=apm-ci/pr-merge
+      - label=automation
+      - files=docker-compose.yml
```

Comment:
These are the conditions:
- If CI is green
- If label `automation`
- If changeset contains `docker-compose.yml`

If you prefer we could include the `dependency` label

OTOH, I could use the author but I need to figure what's the author that the automation will use.

Reply:
I think label=automation is fine.